Drop-In/src/main/java/com/braintreepayments/api/SupportedPaymentMethodsAdapter.java

```diff
@@ -10,27 +11,31 @@
 import android.widget.TextView;
 
 import com.braintreepayments.api.dropin.R;
+import com.braintreepayments.cardform.utils.CardType;
 
 import java.util.List;
 
 class SupportedPaymentMethodsAdapter extends BaseAdapter {
 
-    private final List<DropInPaymentMethodType> supportedPaymentMethods;
-    private final PaymentMethodSelectedListener selectedListener;
+    private final Context mContext;
+    private final List<DropInPaymentMethodType> mAvailablePaymentMethods;
+    private final SupportedPaymentMethodSelectedListener mSupportedPaymentMethodSelectedListener;
 
-    SupportedPaymentMethodsAdapter(List<DropInPaymentMethodType> supportedPaymentMethods, PaymentMethodSelectedListener selectedListener) {
-        this.selectedListener = selectedListener;
-        this.supportedPaymentMethods = supportedPaymentMethods;
+    SupportedPaymentMethodsAdapter(Context context, SupportedPaymentMethodSelectedListener supportedPaymentMethodSelectedListener, List<DropInPaymentMethodType> availablePaymentMethods) {
+        // TODO: remove context variable
```

Comment:
Can context get replaced with `parent.getContext()` throughout this class now?

Reply:
We might be able to, I can't remember if there was a unit test that depends on this but I'll double check.

Reply:
Done!